### ISSUE TITLE
Thumbnail on search() sometimes return object

### ISSUE THREAD

**Issue by KareemH** [VincentJonathan/scrape-yt#5]
I want to store the thumbnail url (which I presume is a string) into my database, and the field of where I store the thumbnail is type String

However, I receive this error
![image](https://user-images.githubusercontent.com/40571613/80545602-c8bf7b00-8981-11ea-8e1c-7627a35b2a1e.png)
![image](https://user-images.githubusercontent.com/40571613/80545632-dd037800-8981-11ea-9922-3919a819e476.png)

Interestingly, some songs are able to store the thumbnail url as a string (coverImage field)......
![image](https://user-images.githubusercontent.com/40571613/80545687-f3a9cf00-8981-11ea-937b-9f8444521945.png)

**Comment by SuspiciousLookingOwl** [comment]
It should return string, i figured out what causes this but i'm not going to release any update for it for now because i'm still working on improving `getVideo()` scrapping.

For now you can do something like this  to check whether thumbnail is string or not
```js
var videos = scrapeYt.search("something");

for (video of videos) {
    var thumbnailUrl = "";
    if (typeof video.thumbnail === "string") {
        thumbnailUrl = video.thumbnail;
    } else {
        thumbnailUrl = video.thumbnail.url
    }
    console.log(thumbnailUrl);
}
```

**Comment by SuspiciousLookingOwl** [comment]
Fixed on 2c504e2fad67195ee51310ffda9b03ea72ccd481
Updated at npm ([v1.0.13](https://www.npmjs.com/package/scrape-yt/v/1.0.13))

Thanks for reporting this!
